spec/lib/ansible/runner_execution_spec.rb

```diff
@@ -13,5 +13,11 @@
       expect(response.return_code).to eq(0), "ansible-runner failed with:\n#{response.stderr}"
       expect(response.human_stdout).to include('"msg": "Hello, world!"')
     end
+    it "runs a hello-world-vault-encrypted playbook" do
+      response = Ansible::Runner.run_vault(env_vars, extra_vars, File.join(__dir__, "runner/data/hello_world_vault_encrypted.yml"))
```

Comment:
Let's try to have the test go through the normal interface.  You should be able to create a vault credential record in the normal way, then pass that credential record in.